src/Ssh/Ssh/Common/SshBaseCmdlet.cs

```diff
@@ -550,18 +577,45 @@ protected internal string GetProxyPath()
         protected internal void GetVmIpAddress()
         {
             string _message = "";
-            Ip = this.IpUtils.GetIpAddress(Name, ResourceGroupName, UsePrivateIp, out _message);
+           
+            (string IpAdress, NetworkInterface networkInterface) = this.IpUtils.GetIpAddress(Name, ResourceGroupName, UsePrivateIp,Bastion, out _message);
```

Comment:
Typo and missing whitespace

```suggestion
            (string IpAddress, NetworkInterface networkInterface) = this.IpUtils.GetIpAddress(Name, ResourceGroupName, UsePrivateIp, Bastion, out _message);
```